lib/Target/AVR/AVRISelLowering.cpp

```diff
@@ -157,6 +157,11 @@ AVRTargetLowering::AVRTargetLowering(AVRTargetMachine &tm)
     // improvements in how we treat 16-bit "registers" to be feasible.
   }
 
+  for (MVT VT : MVT::integer_valuetypes()) {
```

Comment:
I was being lazy originally. I don't *think* that an `i8` needs to have the atomic wrapping as the CPU cannot break it up into pieces.